lib/pure/collections/lists.nim

```diff
@@ -225,7 +225,7 @@ iterator items*[T](L: SomeLinkedList[T]): T =
   ## .. code-block::
```

Comment:
replace with:
```nim
  ## ...
  ## * `nodes iterator <#nodes.i,SomeLinkedList[T]>`_
  runnableExamples:
    from std/sugar import collect
    from std/sequtils import toSeq
    let a = collect(initSinglyLinkedList):
      for i in 1..3: 10*i
    doAssert toSeq(items(a)) == toSeq(a)
    doAssert toSeq(a) == @[10, 20, 30]
```

Reply:
Also those in the `basic usage` part?

Reply:
yes please! `.. code-block::` is not even tested (refs https://github.com/nim-lang/RFCs/issues/309#issuecomment-752821157)